src/app/modules/project/overview/overview.component.scss

```diff
@@ -160,88 +161,92 @@
   }
 }
 
-.projects-filter {
-  box-shadow: 0 3px 6px #0000001a;
-  background: $light-mode-component-color;
-  padding: 20px 25px;
-  border-radius: 6px;
+.projects-filter-wrapper{
+  min-width: 250px;
+  .projects-filter {
+    box-shadow: 0 3px 6px #0000001a;
+    background: $light-mode-component-color;
+    padding: 20px 25px;
+    border-radius: 6px;
+    margin-bottom: 20px;
+    //min-width: 300px;
```

Comment:
I suggest to just delete this line if it's not used 😄

Reply:
Yes sorry stupid me haha

Reply:
No worries 😃 !